### DISCUSSION TITLE
transitioning from postanalysis to exportPlots

### DISCUSSION THREAD

**Discussion by Januaryyiyue** [milaboratory/mixcr#1668] [General]
Hello,

I successfully ran the following command for each of my samples, and got one .json file for each sample
```
mixcr postanalysis individual --default-downsampling count-read-auto --default-weight-function read --metadata /path/to/metadata.tsv --group timepoint /path/to/smaple01_out.clns /path/to/sample01_postanalysis.json --only-productive --drop-outliers
```
How can I run exportPlot on all of them to get one figure with all samples, instead of having one figure per sample (see below)?
<img width="1012" alt="Screenshot 2024-05-22 at 1 08 58 PM" src="https://github.com/milaboratory/mixcr/assets/84147359/a6f343da-de6a-43da-bc0f-8dbaeb17f70b">

**Comment by mizraelson** [comment]
Hi, the command should be run for all .clns files at once. This will generate a JSON file for all samples, which can then be used to export plots. See the example [here](https://mixcr.com/mixcr/guides/generic-umi-bcr/#individual-postanalysis).
